app/Community/Actions/BuildShowForumTopicPagePropsAction.php

```diff
@@ -57,18 +60,46 @@ public function execute(
             hubIds: $entities['hubIds'],
         );
 
+        // Get accessible team accounts for the current user.
+        $accessibleTeamAccounts = null;
+        $accessibleTeamIds = [];
+        if ($user) {
+            $accessibleTeamIds = (new ForumTopicCommentPolicy())->getAccessibleTeamIds($user);
+            if (!empty($accessibleTeamIds)) {
+                $teamUsers = User::whereIn('ID', $accessibleTeamIds)->get();
+                $accessibleTeamAccounts = $teamUsers->map(fn ($teamUser) => UserData::fromUser($teamUser)->include('id'));
+            }
+        }
+
         // Finally, update the message bodies sent to the UI with the converted user shortcodes.
-        $forumTopicComments = $paginatedForumTopicComments->getCollection()->map(function ($comment, $index) use ($updatedBodies) {
-            $comment->body = $updatedBodies[$index];
+        $forumTopicComments = $paginatedForumTopicComments->getCollection()->map(
+            function ($comment, $index) use ($updatedBodies, $user, $accessibleTeamIds) {
+                $comment->body = $updatedBodies[$index];
+
+                $includes = ['user.createdAt', 'user.deletedAt', 'user.visibleRole'];
+
+                /**
+                 * Include the sentBy and editedBy values if:
+                 * A. The user is viewing a team account post they have access to, OR
+                 * B. The user can manage forum posts (they can moderate the forum).
```

Comment:
This enables the edited by functionality for non-team account posts, and there doesn't appear to be any way to specify that the edit occured from a team account.

i.e. As a moderator, I edit a generic user's post to remove a rom link:

<img width="488" height="68" alt="Image" src="https://github.com/user-attachments/assets/4fa083ce-f0e7-4282-8b0f-272a8c94c0ae" />

I think this is a good change, but there should be some way to say the edit was done by RAdmin or something similar.

Reply:
Doesn't doing so obscure who made the edit, though? "Last edited by RAdmin" doesn't seem very helpful if this is supposed to provide some kind of trace of who is editing things.

Reply:
That's definitely what's intended:
```php
/**
 * Include the sentBy and editedBy values if:
 * A. The user is viewing a team account post they have access to, OR
 * B. The user can manage forum posts (they can moderate the forum).
 * If we always naively include it, Inertia will leak the value into the DOM.
 */
$shouldIncludeSentByEditedBy = $user && (
    ($comment->sent_by_id !== null && in_array($comment->author_id, $accessibleTeamIds, true))
    || ($comment->edited_by_id !== null && (new ForumTopicCommentPolicy())->manage($user))
);
```